frontend/app-preview/src/components/PreviewControlHeader/PreviewControlHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { type ReactElement, type ChangeEvent } from 'react';
+import classes from './PreviewControlHeader.module.css';
+import { useTranslation } from 'react-i18next';
+import { useLayoutSetsQuery } from 'app-shared/hooks/queries/useLayoutSetsQuery';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { StudioNativeSelect } from '@studio/components';
+import { ToggleGroup } from '@digdir/designsystemet-react';
+
+export type PreviewControlHeaderProps = {
+  viewSize: 'desktop' | 'mobile';
+  setViewSize: (value: any) => void;
+  selectedLayoutSet: string | null;
+  handleChangeLayoutSet: (value: string) => void;
+};
+
+export const PreviewControlHeader = ({
+  viewSize,
+  setViewSize,
+  selectedLayoutSet,
+  handleChangeLayoutSet,
+}: PreviewControlHeaderProps): ReactElement => {
+  const { t } = useTranslation();
+  const { org, app } = useStudioEnvironmentParams();
+  const { data: layoutSets } = useLayoutSetsQuery(org, app);
+
+  const handleLayoutSetChange = (event: ChangeEvent<HTMLSelectElement>) => {
+    handleChangeLayoutSet(event.target.value);
+  };
+
+  return (
+    <div className={classes.wrapper}>
+      <div className={classes.viewSizeButtons}>
+        <ToggleGroup
+          onChange={setViewSize}
+          value={viewSize === 'desktop' ? 'desktop' : 'mobile'}
```

Comment:
Do we need to check the value of viewSize? It seems like viewSize is either 'desktop' or 'mobile' anyway. 😊 
```suggestion
          value={viewSize}
```

Reply:
This is old code that I've not modified so I had not seen this. I totally agree, and have updated it now 😄